system-tests/README.md

```diff
@@ -1,122 +1,260 @@
-## System tests
+# System tests
+
+The system tests copies a file from a provider to a consumer blob storage account.
+
+## Publish/Build Tasks
+
+> ! Important Note !
+> 
+> MVD depends on Eclipse DataSpaceConnector(EDC), Identity Hub and Registration Service. These dependencies 
+> are __not__ published to any central artifact repository yet, so in local development we have to use locally 
+> published dependencies.
+>
+>In order to use the correct version of each repo required by the `MVD`, you need to look in [action.yml](./.github/actions/../../../.github/actions/gradle-setup/action.yml) for the hashes of the versions of the `EDC`, `Identity Hub` and the `Registration Service` that are being used by the `MVD`.
+>
+> For Example, the `Registration Service`, `Identity Hub` and `EDC` repository hash can be found in the _Checkout_ steps  (in the `ref` property) of [action.yml](./.github/actions/../../../.github/actions/gradle-setup/action.yml):
+
+```yml
+    - name: Checkout EDC
+      uses: actions/checkout@v2
+      with:
+        repository: eclipse-dataspaceconnector/DataSpaceConnector
+        path: DataSpaceConnector
+        ref: 3ff940b720f44826df28e893fb31344eb6faacef
+
+    - name: Checkout Registration Service
+      uses: actions/checkout@v2
+      with:
+        repository: eclipse-dataspaceconnector/RegistrationService
+        path: RegistrationService
+        ref: 374c14bcca23ddb1dcd7476a27264510e54de7fa
+
+    - name: Checkout Identity Hub
+      uses: actions/checkout@v2
+      with:
+        repository: eclipse-dataspaceconnector/IdentityHub
+        path: IdentityHub
+        ref: bc13cf0cb8589b792eef733c7cf7b3422476add5
 
-The test copy a file from provider to consumer blob storage account.
+```
+
+> After you have cloned the `EDC`, `Identity Hub` and `Registration Service` repos locally you should run the command to
+> `reset` to the specific hash.  
+>
+> For Example:
+
+```bash
+# EDC (in the EDC root folder)
+git reset --hard 3ff940b720f44826df28e893fb31344eb6faacef
```

Comment:
```suggestion
git checkout 3ff940b720f44826df28e893fb31344eb6faacef
```
also in other places

Reply:
will make this change but it forces a detached head state.  coming with next PR